odoo/cli/command.py

```diff
@@ -18,31 +18,13 @@ def __init_subclass__(cls):
         cls.name = cls.name or cls.__name__.lower()
         commands[cls.name] = cls
 
+    @classmethod
+    def prog_name(self):
+        return Path(sys.argv[0]).name
 
-ODOO_HELP = """\
-Odoo CLI, use '{odoo_bin} --help' for regular server options.
-
-Available commands:
-    {command_list}
-
-Use '{odoo_bin} <command> --help' for individual command help."""
-
-
-class Help(Command):
-    """ Display the list of available commands """
-    def run(self, args):
-        load_internal_commands()
-        load_addons_commands()
-        padding = max(len(cmd) for cmd in commands) + 2
-        command_list = "\n    ".join([
-            "    {}{}".format(name.ljust(padding), (command.__doc__ or "").strip())
-            for name in sorted(commands)
-            if (command := find_command(name))
-        ])
-        print(ODOO_HELP.format(  # pylint: disable=bad-builtin  # noqa: T201
-            odoo_bin=Path(sys.argv[0]).name,
-            command_list=command_list
-        ))
+    @classmethod
+    def doc(cls):
+        return (cls.__doc__ or "").strip()
```

Comment:
there's a single usage isn't it? I don't think it's worth it to make a method, it can be inlined

Reply:
Nope 
![immagine](https://github.com/user-attachments/assets/7ab7182c-6a3d-4dce-8f43-6125755ae767)

Reply:
Later we can also factor the parser instantiation.

Reply:
I'm fine writing `self.__doc__.strip()` everywhere "explicit > implicit"